### DISCUSSION TITLE
Concern Regarding Missing Changes in SunbirdEd-forms Release

### DISCUSSION THREAD

**Discussion by ramkumar-ns-dev** [Sunbird-Ed/Community#691] [General Discussion]
I'm currently engaged in working on Observation rubrics within the [Sunbird Inquiry editor](https://github.com/Sunbird-inQuiry/editor), particularly focusing on changes related to matrix type questions. In editor we rely on SunbirdEd-forms for these functionalities.

I'm reaching out to address a concern regarding the absence of changes associated with a PR that Pradeep Kumar from our team raised during release-4.9.0. Despite being merged in release-4.9.0, these changes appear to be missing from subsequent releases, including release-4.10.0.

To provide context, here are the details:

PR LINK : https://github.com/Sunbird-Ed/SunbirdEd-forms/pull/173/files
PR merged version : [Sunbird-Ed:release-4.9.0](https://github.com/Sunbird-Ed/SunbirdEd-forms/tree/release-4.9.0)
PR missing version : [Sunbird-Ed:release-4.10.0](https://github.com/Sunbird-Ed/SunbirdEd-forms/tree/release-4.10.1)

Could you please assist in confirming whether this omission was intentional or perhaps an oversight?

Your clarification on this matter would greatly assist our ongoing efforts.

can someone help on this
@vinukumar-vs @rajeevsathish 

cc : @kiranTunerlabs

**Comment by rajeevsathish** [comment]
@vinukumar-vs please have a look on this and if we can take this in the current release of the branch

**Comment by vinukumar-vs** [comment]
@ramkumar-pacewisdom @kiranTunerlabs I dont have the background of this feature. I can noticed it was completly missed in all the following releases of ED.

How is the owner of this feature and why it was not tested in the future releases of ED? 

I can see this as a classic example of unit-tests not writting for the libraries(if it present) or QA has to include the required tests to validate before sing-off on the following releases.

It is too late to escalate or address the issue here. As is it in very old version and from very long time(since 2023) no one reported this as a feature missing.

@mohitga @ashokreddy1208 We can still see some gaps in our process of release merging and QA verification. Let's discuss on this how can we avoid these kind of issues going forward.

**Reply by kiranharidas187** [reply]
@vinukumar-vs These changes are made to enable observation creation capability in the editor. This feature is still in the development phase hence no QA testing happening during releases. Complete details of the changes is provided [here](https://github.com/Sunbird-Ed/SunbirdEd-forms/pull/173)

While the code was merged, the unit test cases were not required, hence it got merged. We can update the component with test cases now.

**Reason for delay in bringing this up** - While setting up dev environment all our configuration and framwork has been deletd by someone from the dev environmnt, it took  a long to for us to bring back all the capabilities and reach to this particular flow. After reaching to this flow only we understood particular code is missing.

Please suggest a solution to bring these code changes to current version rather than redeveloping.
